common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import, print_function
+
+import csv
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from pytz import utc
+from os import remove
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py lms export_staff_users -d 7 --settings=devstack_docker
+        $ ./manage.py lms export_staff_users --days 7 --settings=devstack_docker
+    """
+
+    help = """
+    This command will export a csv of all users who have logged in within the given days and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-d',
+            '--days',
+            type=int,
+            default=7,
+            help='Indicate the login time period in days starting from today'
+        )
+
+    subject = 'Staff users CSV'
+    to_addresses = ['aazam@edx.org']
```

Comment:
should this be an arg as well? It will require the code changes to change the recipient, so we should take care of it beforehand.

Reply:
@asadazam93

Reply:
@DawoudSheraz the email will be sent to the "author-support" group so that is not expected to change.

Reply:
the other thing we can do it to include a `--dry` flag which sends email to `sustaining-mavericks` otherwise it should send email to `author-support`. I am open to other ideas if you have. 

But once that PR lands on PROD, we need to open an additional PR to change the email address from the current.

Reply:
I still think we should add flexibility here.